wsagent/che-core-api-git-shared/src/main/java/org/eclipse/che/api/git/shared/BranchListMode.java

```diff
@@ -10,19 +10,21 @@
  *******************************************************************************/
 package org.eclipse.che.api.git.shared;
 
-import org.eclipse.che.dto.shared.DTO;
-
 /**
- * Request for calling git ls-remote.
+ * Set of branch list modes.
  *
- * @author Vladyslav Zhukovskii
+ * @author Igor Vinokur
  */
-@DTO
-public interface LsRemoteRequest extends GitRequest {
-    /** @return url of remote repository. */
-    String getRemoteUrl();
-
-    void setRemoteUrl(String remoteUrl);
-
-    LsRemoteRequest withRemoteUrl(String remoteUrl);
+public enum BranchListMode {
+    /**
+     * Show both remote and local branches. <br/>
+     * Corresponds to -a option in C git.
+     */
+    LIST_ALL,
+    /**
+     * Show only remote branches. <br/>
+     * Corresponds to -r option in C git.
+     */
+    LIST_REMOTE,
+    LIST_LOCAL
```

Comment:
I would be better to write doc for this enum value if you provide doc for another values